### DISCUSSION TITLE
Datacites roles in nmrXiv

### DISCUSSION THREAD

**Discussion by NRayya** [NFDI4Chem/nmrxiv#630] [Data Schemas]
In order to assign DOIs to the data in nmrXiv, we need to comply with Datacite, our DOI provider. 

Datacite requires providing the creators of the data (The main researchers involved in producing the data/ [field 2 in page 13](https://schema.datacite.org/meta/kernel-4.4/doc/DataCite-MetadataKernel_v4.4.pdf)). They also recommend providing the contributors (The institution or person responsible for collecting, managing, distributing, or otherwise contributing to the development of the resource./ [field 7 in page 19](https://schema.datacite.org/meta/kernel-4.4/doc/DataCite-MetadataKernel_v4.4.pdf)), along with the contributor's type ([page 40](https://schema.datacite.org/meta/kernel-4.4/doc/DataCite-MetadataKernel_v4.4.pdf)).

The challenge we face is to balance between those requirements, keeping the platform simple for the users, and being compatible with other repositories connected to nmrXiv. 

To simplify the platform for the users, it was decided to assign the role of creator by default to every author added to nmrXiv along with the role of researcher. However, if the users are interested in entering specific contributors roles, they can change the researcher to any other contributor role. Currently, only one contributor role can be assigned (along with the creator) so the user needs to give the role with the highest priority.  

For Discussion:

1.  Is there a need to ask the users to manually enter the creators as a field separated from the contributors? 
2. Is there a need to assign more than one contributor role (knowing that the whole contributor field is not even mandatory in Datacite)?

**Comment by tilfischer** [comment]
The DataCite schema knows Creator and the Contributor. For the Contributor there is the contributorType with a controlled list of possible types. However, there is no such list for the Creator.

A Creator might have a creatorName, nameType, givenName, familyName, nameIdentifier and many more but, to my knowledge, there is not such a thing like a "role" for the Creator in DataCite's schema. If it is there, can you please point on on that?

**Comment by tilfischer** [comment]
Example from Chemotion Repository: Sample with ID [CRS-8955](https://dx.doi.org/10.14272/FCAMUPIRWKNASD-UHFFFAOYSA-N.1) has Creators J. Heck, S. Herres-Pawlis and A. Hoffmann without any roles and termed in Chemotion Repository as being "Authors". There is one Contributor, J. Heck, who is a researcher by role.

Example from RADAR: Dataset with DOI https://dx.doi.org/10.22000/613 has all authors of the corresponding publication with DOI https://doi.org/10.1039/D2SC02910C as Creators without any roles and termed "Creator/Author". There is no contributor given for this dataset, who might have a role.

**Comment by tilfischer** [comment]
Please also consider https://github.com/NFDI4Chem/nmrxiv/issues/627, which was closed in favor of this github discussion.

**Comment by vcnainala** [comment]
I believe we should go ahead with the following:

- Creators: All authors added to the project (this is expected to align with the journal/publication if exists or in prep)

- Contributors / Contributor Types: 
         - Authors with their associated roles
         - Submitters of the data to nmrXiv (includes team members and others) with the role of "DataCurator"
                  -  If the submitters have a role already chosen (cases where the submitter is part of the author's list), then the selected role takes precedence over "DataCurator"

**Comment by tilfischer** [comment]
Thank you Chandu for summarising this up. 

I would like to highlight that Chemotion Repository terms "Creators" as "Authors", which might be the better term from the perspective of the users/researchers. To keep connection to DataCite's metadata schema, RADAR people termed this "Creator/Author" to make it instantaneously understandable for users but also still leave the link to the underlying metadata schema visible to users.

**Comment by tilfischer** [comment]
What is the advantage of having all authors listed as creators and have them again listed as contributors? Is this the way how it is also done by other N4C repos? If not, why is this the case?

You wrote that submitters of data to nmrXiv are contributors with the role of a data curator. This does also include all people of a nmrXiv team, if the project was created under the umbrellas of a nmrXiv team. This might be surprising feature for researchers, as this behavior is not obvious.

The question here would be: What is a team? The documentation tells "Teams facilitate collaboration with others on projects by simplifying sharing options." This could be read as all people of a research group might be part of team and this was also the way it was understood e.g. by Steffen. By doing so, some members of a research group might publish their data in nmrXiv under the umbrella of their research group nmrXiv team. They do list themselves as authors, but all other members of the research group, which are also part of the corresponding nmrXiv team, are contributors although they might not have had anything to do with the data published. Listing people there without any contribution might violate the guidelines on good scientific practice.

I would suggest to leave the power and control to the researcher who publish their data in nmrXiv and try to make things absolutely obvious to users/researchers. To make it obvious who is an author/creator or contributor, one might add contributors to the group of things called description, keywords, licenses, citation and authors. Than, everything is on one page, obviously an clearly displayed the the users and the users see what they get.

**Comment by NRayya** [comment]
Hi @tilfischer 

Regarding the team thing we have two options:


1. Our approach to make team members Data curators (as team members usually review each other work). Disadvantage: It is not always the case and users don't have control over it.
2. to ignore the members completely and let the users add contributors as they wish. Good for flexibility but additional work for users. And this will lead back to the question, what should we do with the names with whom a project is shared? where do they fit in DataCite? Initially we went with contributor of type "Other" as they are described: Any person or institution making a
significant contribution to the development and/or maintenance of the resource, Could be a photographer, artist, or writer whose contribution helped to publicize the resource (as opposed to creating it), a reviewer of the resource, someone providing administrative services to the author (such as depositing updates into an online repository, analysing usage, etc.), or one of many other roles.

**Comment by tilfischer** [comment]
Dear Noura, 

leave the power to the people! 

Sure, the questions arises on what to do with team members. Team members should **not** be automatically added as creators or contributors. This might inevitable lead to cases of violation of the guidelines of good scientific practice. 

My suggestion: Do not auto-add team members as creators or contributors, but provide them as suggestions, if one edits the lists of authors somewhere in the panel which opens when clicking "edit" for Authors, see:
![Screenshot 2024-01-03 154546](https://github.com/NFDI4Chem/nmrxiv/assets/85482437/1270b962-27b1-4193-9379-075c50f1c9b6)

Btw: Beside of "Authors" (should be termed "Contributors/Authors" see #948 ), there should be another section for Contributors.

Best,
Tillmann
